src/components/QuizQuestion.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="quiz__container">
+    <div class="quiz__image-box--form" />
```

Comment:
Modifier is used, but there is no `quiz__image-box` class here 🤔

Reply:
I've changed also modifiers' names for `image-box` to be more related to their specificity, not to component itself. If you could please check if it doesn't make more 'noise' than it was.
(`--flexible`, `--fixed`, `--hideable`)